src/applications/vaos/services/var/index.js

```diff
@@ -260,9 +260,12 @@ export function getRequestEligibilityCriteria(sites) {
 }
 
 export function getDirectBookingEligibilityCriteria(sites) {
-  return apiRequestWithMocks(
-    `/vaos/v0/direct_booking_eligibility_criteria?${sites
-      .map(site => `parent_sites[]=${site}`)
-      .join('&')}`,
-  ).then(parseApiList);
+  if (sites) {
```

Comment:
If there's a VAOS test that's failing, feel free to skip that test (after pulling in the latest from master). This code should be reverted back to what it was before, though.

Reply:
ah, nvm, i think i found the failing test to skip:
`/application/src/applications/vaos/tests/appointment-list/components/FutureAppointmentsList.cc.unit.spec.jsx:27:97`

Reply:
nevermind, like every test seems to be failing. this is an issue beyond just one test to skip, as i see it. if i am incorrect, please let me know, otherwise please advise
<img width="1820" alt="Screen Shot 2021-03-10 at 10 26 48 AM" src="https://user-images.githubusercontent.com/46791771/110678546-456ada80-818b-11eb-9f25-eb2ce222ecdd.png">

cc: @mpelzsherman @kelsonic

Reply:
@youngfreezyVA I'm confused. If I look back through Jenkins for this PR, I see one VAOS test that failed one time, which is not the one that you ended up skipping. I don't see any other unit test failures in the Jenkins builds.